src/ui/Base/RegisteredNamedMethods.ts

```diff
@@ -53,6 +53,8 @@ export function initSearchbox(element: HTMLElement, searchPageUri: string, optio
   searchInterfaceOptions.searchPageUri = searchPageUri;
   searchInterfaceOptions.autoTriggerQuery = false;
   searchInterfaceOptions.enableHistory = false;
+  searchInterfaceOptions = { ...searchInterfaceOptions, ...options.StandaloneSearchInterface };
```

Comment:
This is the key change of the PR, allowing someone to pass options to the StandaloneSearchInterface component.

Note that the approach of communicating history between the standalone and main search interface using localstorage will not work if the pages are on different `protocol://host:port`